graphql/util.go

```diff
@@ -59,40 +60,48 @@ func getGroupedFiles(ctx context.Context, name string, taskID string, execution
 	return &GroupedFiles{TaskName: &name, Files: apiFileList}, nil
 }
 
-func setScheduled(ctx context.Context, sc data.Connector, taskID string, isActive bool) (*restModel.APITask, error) {
+func setScheduled(ctx context.Context, sc data.Connector, isActive bool, taskIDs ...string) ([]*restModel.APITask, error) {
```

Comment:
Can we rename this to setManyTasksScheduled ?

Reply:
There are a couple places that are going to pass in a single task, and others where a list is passed in.  I used a spread to handle both cases cleanly instead of converting the single task into an array.